pypeman/msgstore.py

```diff
@@ -74,7 +74,26 @@ async def get_msg_content(self, id):
         :return: A dict `{'id':<message_id>, 'message_content': Message}`.
         """
 
-    async def search(self, start=0, count=10, order_by='timestamp', start_dt=None, end_dt=None):
+    async def is_regex_in_msg(self, id, rtext):
+        """
+        Return True if the str(msg) contains the regex rtext
+
+        :param id: Message id. Message store dependant.
+        :param rtext: String. The rtext is the text to convert in regex and to search in msg
```

Comment:
rtext: string of regular expression  to search in msg

Reply:
done